lib/turbo_boost/commands/runner.rb

```diff
@@ -136,17 +144,16 @@ def prevent_controller_action(error: nil)
       render_response status: response_status
       append_success_to_response
     when TurboBoost::Commands::AbortError
-      render_response status: error.http_status_code, headers: {"TurboBoost-Command-Status": error.message}
+      render_response status: error.http_status_code, status_header: error.message
       append_streams_to_response_body
     when TurboBoost::Commands::PerformError
-      render_response status: error.http_status_code, headers: {"TurboBoost-Command-Status": error.message}
+      render_response status: error.http_status_code, status_header: error.message
       append_error_to_response error
     else
-      render_response status: :internal_server_error, headers: {"TurboBoost-Command-Status": error.message}
+      render_response status: :internal_server_error, status_header: error.message
```

Comment:
I was going to complain about this, but then I saw the status_header usage underneath. It's a shame you sometimes need more context in the diff here.